session_0/word_processing.rb

```diff
@@ -34,11 +38,25 @@ def words_with_prefix(prefix, words)
 # The function `similarity_score` takes two words and returns the
 # similarity score (an integer).
 def similarity_score(word_1, word_2)
-  raise NotImplementedError # TODO
+  counter = 0
+  while word_1[counter] == word_2[counter] && counter < word_1.length && counter < word_2.length;  
```

Comment:
- Writing this over multiple lines would look better
- Since the number of iterations and the increment in each iteration is known, would a for loop be easier to understand?